spec/jobs/rapid_ready_for_decision/disability_compensation_job_spec.rb

```diff
@@ -67,6 +67,9 @@
           Sidekiq::Testing.inline! do
             expect do
               RapidReadyForDecision::DisabilityCompensationJob.perform_async(submission.id)
+
+              submission.reload
```

Comment:
Since `submission.form_json` was updated (and `invalidate_form_hash`) in a different context (a Sidekiq job), we need to tell ruby to reload so that `submission.form_json` is updated in this RSpec context.